### DISCUSSION TITLE
Output of the parsed json?

### DISCUSSION THREAD

**Discussion by BarbWire-1** [ajaxorg/ace#5604] [Ideas]
I'm working on a kind of code explorer/debugger for algorithms from user input. I started with using acron for parsing the code to interpret it.
Now implemented ace - initially for code-highlighting.
I wonder whether there is an accessible json from the parsed code-strucure in ace instead?
(Holding infos of the code-nodes)

**Comment by nightwing** [comment]
Ace does not parse javascript, it uses only a basic tokenizer to highlight the code.

If you are using [ace-linters](https://github.com/mkslanc/ace-linters), you could use typescript compiler from it, but it runs in a web worker so might require restructuring your code.

**Comment by BarbWire-1** [comment]
Thanks for the quick response!
I'll take a look at ace-linters then.

Harutyun Amirjanyan ***@***.***> schrieb am Di., 2. Juli
2024, 09:11:
